src/Board/Boards.js

```diff
@@ -0,0 +1,82 @@
+import React, { useEffect } from "react";
+import { useSelector } from "react-redux";
+import { Link } from "react-router-dom";
+import { useDispatch } from "react-redux";
+import NewBoard from "./NewBoard";
+import { deleteBoard } from "../Data/BoardReducer";
+import useLocalStorage from "local-storage-hook";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faTimes } from "@fortawesome/free-solid-svg-icons";
+import { Card } from "antd";
+import s from "./Boards.module.css";
+
+let Boards = (props) => {
+  const dispatch = useDispatch();
+  const stateBoard = useSelector((state) => state.boards);
+  const [localDataUserBoard, setLocalDataUserBoard] = useLocalStorage(
```

Comment:
I would prefer moving managing localstorage state in redux using something like https://www.npmjs.com/package/redux-persist